Simplified/SignInLogic/NYPLAccountSignInViewController.m

```diff
@@ -461,12 +461,12 @@ - (UIView *)tableView:(UITableView *)__unused tableView viewForFooterInSection:(
     container.preservesSuperviewLayoutMargins = YES;
     UILabel *footerLabel = [[UILabel alloc] init];
     footerLabel.font = [UIFont customFontForTextStyle:UIFontTextStyleCaption1];
-    footerLabel.textColor = [UIColor lightGrayColor];
+    footerLabel.textColor = [NYPLConfiguration fieldTextColor];
```

Comment:
I'm starting to feel like maybe `fieldTextColor` should be renamed to "disabledTextColor"? is there a case where this would be incorrect?

Reply:
I am checking with Marcus if the `fieldTextColor` should only be apply to textfield, or it should be in views like the `Settings` tab. If the latter is what he means, I think we should keep the naming and I will be applying this color in other views.

Reply:
Renamed to `disabledFieldTextColor`